server/graphql/v2/input/ExpenseUpdateInput.ts

```diff
@@ -67,5 +67,9 @@ export const ExpenseUpdateInput = new GraphQLInputObjectType({
       type: LocationInput,
       description: 'The address of the payee',
     },
+    status: {
+      type: GraphQLString,
+      description: 'The status of the expense such as DRAFT, PENDING etc',
+    },
```

Comment:
Why is this needed? The type look wrong but instead of fixing it I'm wondering if this can be removed.

Reply:
This is needed because now we are passing the status as an arg for the `ExpenseUpdateInput` object. Specifically at, 

https://github.com/opencollective/opencollective-frontend/blob/db5ddcfcc0fd0bee15af15d73268116562a509b8/components/expenses/ExpenseForm.js#L117

We can probably remove this if we calculate the `existingExpense` outside of the `if` condition; 

https://github.com/opencollective/opencollective-api/blob/b9cdeb9b9dcc438a4d508769cc76a0a4934110e6/server/graphql/v2/mutation/ExpenseMutations.ts#L138-L143

However I didn't do this as passing it from the front-end seems straightforward and doesn't make an unnecessary db call. Do you have a better alternative? 🤔

Reply:
Can we simply not pass it? 

@Betree what do you think?

Reply:
I don't get why we'd need to pass a `status`, but it definitely looks wrong:
- What does it mean if someone passes `HELLO` as a status? (the current field is not an enum)
- What does it mean if someone passes `PAID` as a status?

`status` is not something we want people to be able to edit, it's an indication of the result from other mutations (`processExpense`). We need to find a better API there.

Reply:
Can't we just omit `draftKey` if it's not a draft?

Reply:
That makes sense. I've refactored the front-end pr to omit the `draftKey` if it's not a draft. Thanks for the idea; that seems such a simple solution. 😉